decidim-comments/spec/types/comment_type_spec.rb

```diff
@@ -91,7 +91,7 @@ module Comments
 
         let(:query) { "{ comments { id } }" }
 
-        it "return comment's comments comments data" do
+        it "return comment's comments data" do
```

Comment:
I'm pretty sure this is right, otherwise it's probably missing an `'`

Reply:
is referring to a comment that has nested comments. So `comment's comments` is the right form.